runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -191,6 +191,9 @@ public class ClueScrollPlugin extends Plugin
 	@Inject
 	private OverlayManager overlayManager;
 
+	@Inject
+	private ChatboxPanelManager chatboxPanelManager;
+
```

Comment:
```suggestion
```
Looks like this was already added also, I guess I dealt with this conflict too and forgot